features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,635 @@
+      | naam                                   | waarde   |
+      | land (13.10)                           | 0000     |
+      | datum aanvang adres buitenland (13.20) | 20150808 |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 000000292                           |
+      | fields              | verblijfplaats.verblijfadres.straat |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+      | naam              | waarde   |
+      | land.code         | 0000     |
+      | land.omschrijving | Onbekend |
+
+    Abstract Scenario: met fields gevraagd om <veld>
+      Gegeven de persoon met burgerservicenummer '000000309' heeft de volgende 'verblijfplaats' gegevens
+      | naam                                       | waarde           |
+      | functieAdres.code (10.10)                  | W                |
+      | datum aanvang adreshouding (10.30)         | 20150808         |
+      | gemeente van inschrijving (09.10)          | 0599             |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                                       | waarde           |
+      | straatnaam (11.10)                         | Boterondiep      |
+      | naam openbare ruimte (11.15)               | Boterondiep      |
+      | huisnummer (11.20)                         | 31               |
+      | postcode (11.60)                           | 3077AW           |
+      | woonplaats (11.70)                         | Rotterdam        |
+      | gemeente_code                              | 0599             |
+      | identificatiecode verblijfplaats (11.80)   | 0599010000208579 |
+      | identificatiecode nummeraanduiding (11.90) | 0599200000219678 |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000309                       |
+      | fields              | <veld>                          |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+      | naam                             | waarde           |
+      | straat                           | Boterondiep      |
+      | huisnummer                       | 31               |
+      | postcode                         | 3077AW           |
+      | woonplaats                       | Rotterdam        |
+      | adresseerbaarObjectIdentificatie | 0599010000208579 |
+      | nummeraanduidingIdentificatie    | 0599200000219678 |
+
+      Voorbeelden:
+      | veld                                |
+      | verblijfplaats.verblijfadres.straat |
+      | verblijfadres.straat                |
+      | straat                              |
```

Comment:
als straat wordt gevraagd moet naamOpenbareRuimte worden geretourneerd

Reply:
Is aangepast.